docs/content/en/getting_started/upgrading.md

```diff
@@ -73,6 +58,8 @@ Replace the first step above with this one: `docker-compose build`
 
 ## Upgrading to DefectDojo Version 2.5.x.
 
+Legacy authorization has been completely removed with version 2.5.0
```

Comment:
should we remind users about the migration that should have happened, but can also be run again using the management command?

Reply:
With this PR they won't have the chance anymore to use the management command to migrate the users. They would have to do it before they update to release 2.5.0. Is this what you want to say? Would definitely make sense.

Reply:
Just had the feeling it was good to bring it up here, but didn't yet realize it would have to run before the upgrade. Should be mentioned as (possible) breaking change then I guess.

Reply:
Have added a remark to the release notes.